synapse/push/push_tools.py

```diff
@@ -51,8 +37,10 @@ async def get_room_unread_count(room_id: str) -> None:
             # return one badge count per conversation
             badge += 1
         else:
-            # increment the badge count by the number of unread messages in the room
+            # Increase badge by number of notifications in room
+            # NOTE: this includes threaded notifications as well as non-threaded
```

Comment:
To match the wording in the spec:

```suggestion
            # NOTE: this includes threaded and unthreaded notifications.
```